package.json

```diff
@@ -60,34 +63,46 @@
     "eslint": "^4.12.1",
     "eslint-config-satya164": "^1.0.1",
     "eslint-plugin-react-native-globals": "^0.1.0",
-    "husky": "^0.14.3",
+    "husky": "^1.1.2",
     "jest": "^22.1.3",
     "jest-expo": "^30.0.0",
     "prettier": "^1.8.2",
     "react": "16.3.1",
     "react-dom": "16.3.1",
     "react-native": "^0.55.4",
     "react-test-renderer": "16.3.1",
-    "release-it": "^7.6.1"
+    "release-it": "^7.6.1",
+    "ts-jest": "^23.10.4",
+    "typescript": "^3.1.3"
   },
   "peerDependencies": {
     "react": "*"
   },
   "jest": {
     "preset": "react-native",
-    "testRegex": "/__tests__/[^/]+-test\\.js$",
+    "testRegex": "/__tests__/[^/]+-test\\.[jt]sx?$",
```

Comment:
imo this is more readable:

```suggestion
    "testRegex": "/__tests__/[^/]+-test\\.(js|tsx?)$",
```

Reply:
100% agree. gonna change this! :)

Reply:
one question, this will not catch a `ts` file right? so I guess the correct regex would be:

```js
"testRegex": "/__tests__/[^/]+-test\\.(js|ts|tsx)?$",
```

or the last `?` treat the `x` as optional?